src/components/structures/UserSettings.js

```diff
@@ -921,6 +921,25 @@ module.exports = React.createClass({
         </div>;
     },
 
+    _renderTermsAndConditionsLinks: function() {
+        if (SdkConfig.get().terms_and_conditions_links) {
+            const tncLinks = [];
+            for (const tncEntry of SdkConfig.get().terms_and_conditions_links) {
```

Comment:
`map` would fit well here? But oh well ...

Reply:
yeah true - I suppose it doesn't make a lot of difference